### PR TITLE
Add testing with Ruby 2.7

### PR DESCRIPTION
**Pull request by aleksejleonov**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,6 +16,8 @@ jobs:
           - ruby:2.5
           - ruby:2.6.0
           - ruby:2.6
+          - ruby:2.7.0
```
Comment (pointlessone) [inline .github/workflows/ci.yml]: I'd rather not add this one yet. This is effectively the same as `2.7` just adds to the build time.

**Comment by pointlessone** [comment]
Thank you for your contribution. I'll hold off merging it for a bit. `rc` is the 2.7. It's the last Release Candidate which is nearly identical to the final 2.7.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (aleksejleonov) [reply .github/workflows/ci.yml]: It's more about be prepared for future patch versions (`2.7.1`, etc.) that usually happens quite fast after initial release :)
Also, build matrix jobs works in parallel, so should not take additional time.

**Comment by pointlessone** [comment]
Thank you for your contribution.

It's been a while and our CI setup changes since then. We now support 2.7 and Ruby 3.

**Pull request closed without merging**